ims/common/config.py

```diff
@@ -9,53 +9,94 @@
 __config = None
```

Comment:
I dont see how will this work ?

Method will create a new object which we dont want.

A Class will need an object that needs to be global.